pkgs/applications/networking/ipfs/default.nix

```diff
@@ -1,26 +1,35 @@
-{ stdenv, buildGoPackage, fetchFromGitHub, fetchgx }:
+{ stdenv, buildGoPackage, fetchFromGitHub, fetchgx
+, makeWrapper, fuse
+}:
 
 buildGoPackage rec {
   name = "ipfs-${version}";
-  version = "0.4.10";
-  rev = "4679f806bd00c0a5299c22c82d1fbfdbad928e6d";
+  version = "0.4.11-pre";
```

Comment:
What's the reason to package an unreleased version? We generally try to use releases with patches to fix specific issues if needed.

Reply:
There's no pressing need for any new features, so I'll remove this commit.